src/gateways/storage/index.ts

```diff
@@ -27,6 +27,12 @@ export async function getAllUsers(): Promise<UserRecord[]> {
   return pg('users').select('*')
 }
 
+export async function getUser(id: string | undefined): Promise<UserRecord> {
```

Comment:
I don't think this method should accept `undefined` for the `id`, it makes the interface quite weird. Also, what happens when this is called with `undefined`?

Reply:
If you decide to change the returned columns to take care of the security issues I commented about, don't forget to update the type returned by this function. You might want to use [`Pick`](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktk) or [`Omit`](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk) for that.